invenio_vocabularies/contrib/subjects/services.py

```diff
@@ -9,9 +9,21 @@
 
 """Subjects services."""
 
+from invenio_db import db
 
+from ...records.models import VocabularyScheme
 from .subjects import record_type
 
 SubjectsServiceConfig = record_type.service_config_cls
 
-SubjectsService = record_type.service_cls
+
+class SubjectsService(record_type.service_cls):
+    """Subjects service."""
+
+    def create_scheme(self, identity, id_, name="", uri=""):
```

Comment:
More of a convenience for the subjects service than a necessity if we go with supporting schemes more generally (i.e. affiliations would have to implement its own... meh)